socorro/tests/external/es/test_crashstorage.py

```diff
@@ -109,7 +109,7 @@ def test_index_crash_indexable_keys(self, es_helper):
             "another_invalid_key": "alpha",
             "date_processed": date_to_string(utc_now()),
             "uuid": "936ce666-ff3b-4c7a-9674-367fe2120408",
-            "dom_fission_enabled": "1",
+            "dom_fission_enabled": False,
```

Comment:
We changed this from `"1"` (a true-ish value) to `False`. Was there a reason for that? Does it help the tests?

Reply:
probably also early debugging copypasta, reverting...